src/main/java/util/events/ModelUpdatedEvent.java

```diff
@@ -1,20 +1,19 @@
 package util.events;
 
-import backend.interfaces.IModel;
-import backend.resource.MultiModel;
-import backend.resource.TurboIssue;
+import backend.resource.TurboUser;
 import filter.expression.FilterExpression;
-import org.apache.commons.lang3.tuple.ImmutablePair;
+import ui.GuiElement;
 
 import java.util.List;
 import java.util.Map;
 
 public class ModelUpdatedEvent extends Event {
-    public final IModel model;
-    public final Map<FilterExpression, List<TurboIssue>> issuesToShow;
+    public final Map<FilterExpression, List<GuiElement>> elementsToShow;
+    public final List<TurboUser> users;
```

Comment:
Maybe we should have the comments on what these are for here instead, where the fields are, rather than at the places where this object is constructed. That would put them more in context and also not have them repeated.